lib/solarwinds_apm/support/service_key_checker.rb

```diff
@@ -16,6 +16,7 @@ def initialize(reporter)
 
     def read_and_validate_service_key
       return '' unless @reporter == 'ssl'
+      return '' unless ENV['LAMBDA_TASK_ROOT'].to_s.empty? && ENV['AWS_LAMBDA_FUNCTION_NAME'].to_s.empty? # lambda env doesn't need SW_APM_SERVICE_KEY
```

Comment:
Here again would be nice if there's a single `lambda?` that can be called anywhere.

Reply:
Updated